python/tests/test_shared_pipeline_stress.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+from tests.shared_test_pipeline import SharedTestPipeline
+from tests import TEST_CLIENT
+
+
+class TestPipeline(SharedTestPipeline):
+    result = None
```

Comment:
This shouldn't be necessary in this test setup. Its just a workaround in the other test.

Reply:
what exactly? useing Shared pipeline or result = None?

Reply:
assuming the later.